src/Chapter2.hs

```diff
@@ -621,7 +625,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x:_:xs) = x : takeEven xs
+takeEven (x:_) = x : takeEven []
```

Comment:
I would write this case more explicitly to be sure that the compiler can check all patterns :slightly_smiling_face: Also, since `takeEven []` returns `[]`, you can simplify this as well
```suggestion
takeEven [x] = [x]
```